src/sagemaker/fw_utils.py

```diff
@@ -55,6 +55,7 @@
 VALID_PY_VERSIONS = ["py2", "py3"]
 VALID_EIA_FRAMEWORKS = ["tensorflow", "tensorflow-serving", "mxnet", "mxnet-serving"]
 VALID_ACCOUNTS_BY_REGION = {"us-gov-west-1": "246785580436", "us-iso-east-1": "744548109606"}
+DEBUGGER_FEATURE_UNSUPPORTED_REGIONS = ["us-gov-west-1", "us-iso-east-1"]
 ASIMOV_VALID_ACCOUNTS_BY_REGION = {"us-iso-east-1": "886529160074"}
```

Comment:
it feels weird to split up `VALID_ACCOUNTS_BY_REGION` and `ASIMOV_VALID_ACCOUNTS_BY_REGION`. I'd actually move this new constant to be separate from all the constants that exist for determining a Docker image URI, as this new one has nothing to do with that

Reply:
+1. Fixed